### PR TITLE
Ireland html

### PR DESCRIPTION
**Pull request by lcurtis4**
## Changes made
Added html for Ireland 

## Steps to test
Launch to cover page 
click on Ireland tab in nav bar
see that all images and text needed is there

### CODE REVIEW

File: ireland.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Nashville Software School</title>
```
Comment (cslunetta) [inline ireland.html]: Change title to reflect what the page content is.

```suggestion
        <title>Hello World: Ireland</title>
```

File: ireland.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+        <main id="container">
+            <article class="lists">
+                <section class="lists__cities">
+                    <ul>
+                        <li>Dublin</li>
+                        <li>Belfast</li>
+                        <li>Cork</li>
+                    </ul>
```
Comment (Chris-Denmark) [inline ireland.html:32]: You'll want to add an h2 header for your list here.
Example: Cities To Visit
```suggestion
                <h2> Cities To Visit </h2>
                    <ul>
                        <li>Dublin</li>
                        <li>Belfast</li>
                        <li>Cork</li>
                    </ul>
```

File: ireland.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                </section>
+                <section class="lists__landmarks">
+                    <ul>
+                        <li>Rock of Cashel</li>
+                        <li>Newgrange Tomb</li>
+                        <li>Blarney Stone and Castle</li>
+                    </ul>
```
Comment (Chris-Denmark) [inline ireland.html:40]: You'll want to add an h2 header for your list here.
Example: Famous Landmarks

```suggestion
                <h2>Famous Landmarks</h2>
                    <ul>
                        <li>Rock of Cashel</li>
                        <li>Newgrange Tomb</li>
                        <li>Blarney Stone and Castle</li>
                    </ul>
```

File: ireland.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                </section>
+                <section class="lists__citizens">
+                    <ul>
+                        <li>Micheal Collins</li>
+                        <li>Maureen O' Hara</li>
+                        <li>Katie Taylor</li>
+                    </ul>
```
Comment (Chris-Denmark) [inline ireland.html:48]: You'll want to add an h2 header for your list here.
```suggestion
                <h2> Famous Citizens </h2>
                    <ul>
                        <li>Micheal Collins</li>
                        <li>Maureen O' Hara</li>
                        <li>Katie Taylor</li>
                    </ul>
```
Example: Famous Citizens

**Review by Chris-Denmark — CHANGES_REQUESTED** [review]

File: ireland.html (hunk 3 above)
Comment (cslunetta) [reply ireland.html:40]: updated previous comment to show recommended code.

**Comment by Chris-Denmark** [comment]
👍

**Comment by cslunetta** [comment]
👍

**Pull request merged**